bluepyemodel/__init__.py

```diff
@@ -1,7 +1,7 @@
 """Main module of BluePyEModel."""
 
 """
-Copyright 2023, EPFL/Blue Brain Project
+Copyright 2024, EPFL/Blue Brain Project
```

Comment:
and same for all the copyrights, I won't flood you with a comment for each of them